node.gyp

```diff
@@ -1125,6 +1125,13 @@
             },
           ],
         }],
+        [ 'OS=="linux"', {
+          'link_settings': {
+            'libraries': [
+              '-lbpf',
```

Comment:
Not sure if that's desired as we won't have a conditional if the system doesn't have libbpf installed.